internal/util/render/executor.go

```diff
@@ -193,14 +211,14 @@ type pkgNode struct {
 }
 
 // newPkgNode returns a pkgNode instance given a path or pkg.
-func newPkgNode(path string, p *pkg.Pkg) (pn *pkgNode, err error) {
+func newPkgNode(path string, p *pkg.Pkg, fs filesys.FileSystem) (pn *pkgNode, err error) {
```

Comment:
nit: make `fsys` as the first argument and also use `fsys` name to differentiate from `fs.FS`